contrib/org.freedesktop.fwupd.json

```diff
@@ -0,0 +1,185 @@
+            "--disable-keymap",
+            "--disable-mtd_probe"
+          ],
+          "cleanup": [
+            "/etc/udev",
+            "/libexec/accelerometer",
+            "/libexec/ata_id",
+            "/libexec/cdrom_id",
+            "/libexec/collect",
+            "/libexec/firmware",
+            "/libexec/rules.d",
+            "/libexec/scsi_id",
+            "/libexec/udevd",
+            "/libexec/v4l_id",
+            "/share/gtk-doc/html/libudev/",
+            "/sbin/udevadm"
+          ],
+          "sources": [
+            {
+                "type": "archive",
+                "url": "http://kernel.org/pub/linux/utils/kernel/hotplug/udev-175.tar.bz2",
+                "sha256": "4c7937fe5a1521316ea571188745b9a00a9fdf314228cffc53a7ba9e5968b7ab"
+            },
+            {
+                "type": "script",
+                "dest-filename": "autogen.sh",
+                "commands": [
+                    "autoreconf -vfi"
+                ]
+            }
+          ]
+        },
+        {
+            "name": "libusb",
+            "config-opts": [
+                "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://sourceforge.net/projects/libusb/files/libusb-1.0/libusb-1.0.21/libusb-1.0.21.tar.gz",
+                    "sha256": "1a5b08c05bc5e38c81c2d59c29954d5916646f4ff46f51381b3f624384e4ac01"
+                }
+            ]
+        },
+        {
+            "name": "gusb",
+            "buildsystem": "meson",
+            "config-opts": ["-Ddocs=false",
+                            "-Dvapi=false",
+                            "-Dtests=false"],
+            "cleanup": [
+                "/bin/gusbcmd"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://people.freedesktop.org/~hughsient/releases/libgusb-0.3.0.tar.xz",
+                    "sha256": "d8e7950f99b6ae4c3e9b8c65f3692b9635289e6cff8de40c4af41b2e9b348edc"
+                }
+            ]
+        },
+        {
+            "name": "efivar",
+            "buildsystem": "simple",
+            "build-commands": ["make prefix=/app libdir=/app/lib", "make install prefix=/app libdir=/app/lib"],
+            "config-opts": [""],
+            "cleanup": [
+                "/bin/efivar"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/rhboot/efivar/releases/download/35/efivar-35.tar.bz2",
+                    "sha256": "1e033dc5d099a44fd473b0887dbcc4b105613efab0fb3c5df9f111ea5d147394"
+                }
+            ]
+        },
+        {
+            "name": "fwupdate",
+            "buildsystem": "simple",
+            "build-commands": ["make -C linux prefix=/app libdir=/app/lib", "EFIDIR=/boot/efi make -C linux install prefix=/app libdir=/app/lib"],
```

Comment:
I think we'll need to check and make sure EFIDIR=/boot/efi works from flatpak.  If not, we might need some extra plumbing for what to do with these EFI directories when running containerized.

Reply:
Oh good.